frontend/src/components/RegisterForm.tsx

```diff
@@ -29,10 +29,6 @@ function RegisterForm(props: RegisterFormProps) {
       errors.email = "Please enter your email";
       hasError = true;
     }
-    if (!isPasswordValid(values.password1)) {
-      errors.password1 = "Password must have at least 8 chars and 1 number";
-      hasError = true;
-    }
     if (values.password2 !== values.password1) {
```

Comment:
Are we hashing passwords?

Reply:
Yes, we hash them server side with argon2.